From 6f1cb53516f1440a91cc1b556ce223e36b95bb30 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@frodo>
Date: Sat, 3 Aug 2019 07:30:47 +0200
Subject: [PATCH] Revert "Only build LOKDocView-0.1.gir when necessary"

Only building it with PKFORMAT deb/rpm effectively disables it for
make install/disto-pack-install...

This reverts commit 4b4c7e76e6c2ea0d47a42a5107352d3ad7341fbf.
---
 sysui/CustomTarget_share.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sysui/CustomTarget_share.mk b/sysui/CustomTarget_share.mk
index 9afe6da42a45..231aa4ff930d 100644
--- a/sysui/CustomTarget_share.mk
+++ b/sysui/CustomTarget_share.mk
@@ -127,9 +127,9 @@ $(eval $(call gb_CustomTarget_register_targets,sysui/share,\
 		$(product)/openoffice.keys \
 		$(product)/openoffice.sh \
 		$(product)/create_tree.sh \
-		$(if $(filter deb rpm,$(PKGFORMAT)),$(if $(INTROSPECTION_SCANNER),\
+		$(if $(INTROSPECTION_SCANNER),\
 			$(product)/LOKDocView-0.1.gir \
-			$(product)/LOKDocView-0.1.typelib)) \
+			$(product)/LOKDocView-0.1.typelib) \
 		$(product)/launcherlist) \
 ))
 
-- 
2.20.1

